docs/intro/install.txt

```diff
@@ -57,12 +57,15 @@ You've got three easy options to install Django:
   who have operating systems that distribute Django.
 
 * :ref:`Install an official release <installing-official-release>`. This
-  is the best approach for users who want a stable version number and aren't
-  concerned about running a slightly older version of Django.
+  is the best approach for users who want the latest stable release.
```

Comment:
This feels a bit circular. "The latest official release is the best approach for users who want the latest stable release." Ok, so we've learned that "official -> stable" :-)

I would be tempted to simplify even further: "This is the best approach for most users."